README.md

```diff
@@ -1,55 +1,70 @@
 # cfgov-refresh
 
-This repository contains the redesign-in-progress of consumerfinance.gov. This includes front-end assets and build tools, and configuration for [Sheer](https://github.com/cfpb/sheer) to load content from Wordpress and Django back-ends to elasticsearch to render the site.
+**Description**: This repository contains the redesign-in-progress of the [consumerfinance.gov](http://consumerfinance.gov) website. This includes front-end assets and build tools, and configuration for [Sheer](https://github.com/cfpb/sheer) to load content from WordPress and Django back-ends to Elasticsearch to render the site.
 
-### This project is a work in progress
+**Technology stack**:
```

Comment:
The nice thing about a vagrant devbox is that it can be used for reference for writing Unix installation instructions and then given as a Windows installation, see https://github.com/codeforamerica/ohana-api-dev-box for reference.

Pulling the instructions into a separate `INSTALL.md` file can also be nice. Instructions can take a huge amount of space in a readme, so can hurt scan-ability for reading other sections (see https://github.com/codeforamerica/ohana-api).

Generalized instructions seem fine, but I think there needs to be team-specific on-boarding instructions somewhere if that isn't covered in the Readme. Perhaps the readme could be team-specific (OSX for this repo), with a side-note for Unix differences? I think that just means providing an alternative instruction path when Brew isn't available.